### ISSUE TITLE
What does the geo-political entity mean?

### ISSUE THREAD

**Issue by Jingyilang** [hltcoe/golden-horse#14]
Thanks for your data. However, I can't understand the  geo-political entity. It seems like that  geo-political  is same as  LOCATION. Can you tell me more about the geo-political entity? Thanks.

**Comment by wfyhehe** [comment]
Hi @Jingyilang , I'm experimenting with these data with my model. For me, I reckon that the GPE stands for names of *countries, provinces, cities, etc* like `北京, 广东, 中国, 旧金山..`, having something to do with *politics*. While the LOC stands for names of places on a smaller scale, e.g. `长安街, 人民广场, 如家酒店...`
However, I noticed that `丽江，吐鲁番`(names of cities in China) are labeled as LOC, and I'm a bit confused too. I guess it's that `丽江` is mistakenly considered as a river(江), and `吐鲁番` is not like other city names because it got its name from another language other than Chinese.

**Comment by VioletPeng** [comment]
Indeed, GPE stands for names of countries, provinces, cities, etc like 北京, 广东, 中国, 旧金山.
It is very similar to LOC so sometimes people will make mistake.
